addons/knowledge/views/knowledge_views.xml

```diff
@@ -0,0 +1,247 @@
+                                    <div attrs="{'invisible': ['|', ('is_locked', '=', False), ('user_can_write', '=', False)]}">
+                                        <i class="fa fa-fw fa-lock" title="locked"/>
+                                    </div>
+                                </div>
+                                <div class="flex-grow-1 text-muted">
+                                    <field name="category"/>
+                                </div>
+                                <div class="d-flex justify-content-end align-items-center">
+                                    <field name="activity_ids" widget="kanban_activity"/>
+                                </div>
+                            </div>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
+
+    <record id="knowledge_article_view_search" model="ir.ui.view">
+        <field name="name">knowledge.article.view.search</field>
+        <field name="model">knowledge.article</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name"/>
+                <field name="main_article_id"/>
+                <field name="last_edition_uid"/>
+
+                <filter string="Archived" name="active" domain="[('active', '=' ,False)]"/>
+                <filter name="favourites" string="My Favourites" domain="[('is_user_favourite', '=', True), ('user_has_access', '=', True)]"/>
+                <filter name="own_articles" string="My Articles" domain="[('owner_id', '=', uid), ('user_has_access', '=', True)]"/>
+                <filter name="shared_articles" string="My Shared Articles" domain="[('category', '=', 'shared'), ('user_has_access', '=', True)]"/>
+
+                <group expand="0" string="Group By">
+                  <filter string="Category" name="category" domain="[]" context="{'group_by':'category'}"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="knowledge_article_action">
+        <field name="name">Articles</field>
+        <field name="res_model">knowledge.article</field>
+        <field name="view_mode">tree,kanban,form</field>
+        <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">
+                Create an article
+            </p><p>
+                Be the first one to unleash the power of Knowledge !
+            </p>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="knowledge_article_dashboard_action">
+        <field name="name">Articles</field>
+        <field name="res_model">knowledge.article</field>
+        <field name="view_mode">form</field>
+        <field name="view_id" ref="knowledge.knowledge_article_view_dashboard"/>
+    </record>
+
+    <record id="action_home_page" model="ir.actions.server">
```

Comment:
I see in website that they apply some prefix naming to this: "ir_actions_server_website_dashboard".